quelpa.el

```diff
@@ -472,6 +479,8 @@ the global var `quelpa-upgrade-p' is set to nil."
   (when (quelpa-setup-p)
     ;; shadow `quelpa-upgrade-p' taking the default from the global var
     (let* ((quelpa-upgrade-p (if current-prefix-arg t quelpa-upgrade-p))
+           ;; shadow `quelpa-stable-p'
+           (quelpa-stable-p quelpa-stable-p)
```

Comment:
Oh right, let-binding them before allows the trick to work.  I still find it pretty convoluted compared to just using `or` on the global value and a plist retrieval though.